docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -212,10 +212,44 @@ \section{Safety and Security Requirements}
 \wss{Newly discovered requirements.  These should also be added to the SRS.  (A
 rationale design process how and why to fake it.)}
 
+\subsubsection*{NFR-SE4}
+\begin{itemize}
+  \item \textbf{Description:} The application shall prohibit new user actions when user is disconnected from the system.
+  \item \textbf{Rationale:} The system should avoid batch updates upon reconnect as to prevent malicious users creating a false package of updates.
+  \item \textbf{Fit Criterion:} Upon user disconnect and reconnect, the system will not accept any updates from the user not submitted before the disconnect occured.
+\end{itemize}
+
+\subsubsection*{NFR-SE6}
```

Comment:
Should we add another Requirement around re requesting data from API/backend when a request fails? Something about if we get an error response back we try again after x seconds/minutes etc. I think this would go well with H4-1 and H4-2 as well